ajax/libs/autonumeric/package.json

```diff
@@ -1,37 +1,51 @@
 {
   "name": "autonumeric",
-  "version": "1.9.46",
-  "description": "autoNumeric is a jQuery plugin that automatically formats currency (money) and numbers as you type on form inputs. It supports most International numeric formats and currency signs including those used in Europe, North and South America, Asia and India's' lakhs",
-  "filename": "autoNumeric.min.js",
+  "version": "4.0.0-beta.2",
+  "description": "autoNumeric is a standalone Javascript library that provides live *as-you-type* formatting for international numbers and currencies. It supports most International numeric formats and currencies including those used in Europe, Asia, and North and South America.",
+  "filename": "dist/autoNumeric.min.js",
   "keywords": [
     "currency",
     "money",
+    "monetary",
     "Euro",
     "Dollar",
     "Pound",
     "number",
     "numeric",
     "format",
-    "jquery",
     "form",
     "input",
-    "mask"
+    "mask",
+    "as-you-type",
+    "live"
   ],
   "license": "MIT",
+  "authors": [
+    {
+      "name": "Robert Knothe",
+      "email": "bob@decorplanit.com"
+    },
+    {
+      "name": "Alexandre Bonneau",
+      "email": "alexandre.bonneau@linuxfr.eu"
+    }
+  ],
   "homepage": "http://www.decorplanit.com/plugin/",
   "repository": {
     "type": "git",
-    "url": "git://github.com/BobKnothe/autoNumeric.git"
+    "url": "git+https://github.com/BobKnothe/autoNumeric.git"
   },
-  "author": "Robert Knothe",
   "autoupdate": {
     "source": "git",
     "target": "git://github.com/BobKnothe/autoNumeric.git",
     "fileMap": [
       {
         "basePath": "",
         "files": [
-          "autoNumeric.js"
+          "src/AutoNumeric.js",
+          "src/AutoNumericEnum.js",
+          "src/AutoNumericHelper.js",
+          "src/main.js"
```

Comment:
hey @pvnr0082t , I'm not sure to follow what you meant by *"Let's use `"**/*"` instead of these rules"*.

Do you mean replacing those lines by
```js
          "src/*.js",
// or
          "**/*.js",
```
?